wazo_export_import/helpers/constants.py

```diff
@@ -215,4 +215,16 @@
         ],
         "references": ["destination", "schedule"],
     },
+    "fk": {
```

Comment:
Be more explicit in definition
```suggestion
    "function_keys": {
```
(but no objection to only use `fk` in variable name)

Reply:
Fixed, I removed the `mergeit` tag until I complete a full export/import with the modifications. I'll add the `mergeit` tag with it succeeds.